ui/src/app/routing/guards.tsx

```diff
@@ -16,7 +17,7 @@ export const disabledGuard = (userDisabled: boolean): Guard => ({
 
 export const registrationGuard: Guard = {
   allowed: (): boolean => hasRegisteredAccess(profileStore.get().profile.accessTierShortNames),
-  redirectPath: '/'
+  redirectPath: environment.enableDataAccessRequirements ? '/data-access-requirements' : '/'
```

Comment:
FF = false (current behavior) -> unregistered users are redirected to the homepage which **includes** the Registration Dashboard

FF = true -> unregistered users are redirected to Data Access Requirements which is **separate** from the homepage